### PR TITLE
remove isInsideInterface

### PR DESCRIPTION
**Pull request by LoiNguyenCS**
Professor,

Turns out that the field `isInsideAnInterface` was buggy; once set to `true`, it remained permanently in that state. So I have removed that field. I have also fixed a test file we had in the past, where a used method should have an empty body.

### CODE REVIEW

File: src/test/resources/MethodInEnum/expected/com/example/Foo.java [hunk 1]
```diff
@@ -18,9 +18,7 @@ public static void testing() {
         }
         ;
 
-        public abstract void testing() {
-            throw new Error();
-        }
+        public abstract void testing();
```
Comment (LoiNguyenCS) [inline src/test/resources/MethodInEnum/expected/com/example/Foo.java:21]: This one was originally a method signature. The old version of Specimin added a `throw new Error()` body by mistake.
